corehq/ex-submodules/casexml/apps/phone/restore_caching.py

```diff
@@ -48,7 +49,8 @@ def _get_new_arbitrary_value():
     return datetime.datetime.utcnow().isoformat()
 
 
-@quickcache(['domain'], timeout=60 * 24 * 60 * 60)
+@quickcache(['domain'], timeout=60 * 24 * 60 * 60,
+    session_function=icds_conditional_session_key(), memoize_timeout=60 * 60)
 def _get_domain_freshness_token(domain):
```

Comment:
@snopoke Any thoughts here?

Reply:
Yes, I agree that it's usefulness is minimal. I guess it's one of those things that you may want one day. I think it's worth keeping this one but not the per user one. My thinking is that if for some reason we need to invalidate the cache for ALL users then it will be really hard to do without this. If we're just doing a few users then I think we could do it manually.

We could also get rid of `get_loadtest_factor_for_user` on ICDS.